charts/actions-runner-controller/values.yaml

```diff
@@ -26,7 +26,7 @@ authSecret:
   #github_token: ""
 
 image:
-  repository: summerwind/actions-runner-controller
+  repository: actions-runner-controller/actions-runner-controller
```

Comment:
Same as above, not sure that one needs changing?

Reply:
@sledigabel Ah, sorry I missed your comments! Thanks- I'm fixing the two issues in #663